wagtailcache/cache.py

```diff
@@ -184,6 +195,33 @@ def clear_cache() -> None:
     if wagtailcache_settings.WAGTAIL_CACHE:
         caches[wagtailcache_settings.WAGTAIL_CACHE_BACKEND].clear()
 
+def clean_uri(uri):
```

Comment:
I'm thinking this should be a "private" method... in the event that we need to change how it works (e.g. there might be some scenarios in the future where it will need to parse querystrings, etc.)

```python
def _clean_uri(uri):
```